liiatools_pipeline/ops/common_la.py

```diff
@@ -129,6 +126,8 @@ def process_files(
         try:
             schema = globals()[f"load_schema_{config.dataset}"](year)
         except KeyError:
+            log.warning(f"Unable to load schema: {config.dataset} from year {year}")
+            # ISSUE: It will likely be an issue that schema is undefined past this point, right?
```

Comment:
@patrick-troy Any idea what I could set schema to if the above line fails?  It's important that we validate and catch issues with things like schemas, input files, and configurations as they can easily have mistakes in them potentially and it's better to catch them than to allow them to fail and escalate. That's the reason for my putting this in a try/except clause here..

Reply:
If we can't get a correct schema I don't think we should process the file as we can't be sure how to clean it properly. I didn't catch the error using `except KeyError` but the following code I recommend. The reason for the `continue` is so that the process does not clean the file it can't find a schema for, but will still clean the other files from that folder.

`try:`
`    schema = globals()[f"load_schema_{config.dataset}"](year)`
`except Exception as err:`
`    log.error(f"Unable to load schema: {config.dataset} from year {year} error: {err}")`
`    continue`

When running this the error I get is: <urlopen error [WinError 2] The system cannot find the file specified: ....> so we can be more specific about catching this error if necessary

Reply:
There are two potential errors here: